packages/react-dom/src/__tests__/ReactDOMHydrationDiff-test.js

```diff
@@ -1063,27 +1063,18 @@ describe('ReactDOMServerHydration', () => {
             </div>
           );
         }
-        // TODO: Why does this not show a fallback mismatch?
-        // And why is this message different from the other ones?
         if (
           gate(flags => flags.enableClientRenderFallbackOnHydrationMismatch)
         ) {
-          expect(testMismatch(Mismatch)).toMatchInlineSnapshot(`
-            Array [
-              "Caught [The server could not finish this Suspense boundary, likely due to an error during server rendering. Switched to client rendering.]",
-            ]
-          `);
+          // There is no error because we don't actually hydrate fallbacks.
```

Comment:
Do you have other tests that cover the case when props flow into the boundary which would still trigger this case?

Reply:
Added in https://github.com/facebook/react/pull/24236/commits/e6734cdcde30dba460b7b0b7227ef811443f4f03.